packages/httpsnippet-client-api/test/index.test.ts

```diff
@@ -111,7 +110,7 @@ describe('httpsnippet-client-api', () => {
       it('should generate the expected snippet', async () => {
         const expected = await fs.readFile(path.join(DATASETS_DIR, snippet, 'output.js'), 'utf-8');
 
-        const code = new HTTPSnippet(mock.har).convert('node', 'api', {
+        const code = await new HTTPSnippet(mock.har).convert('node', 'api', {
```

Comment:
Thanks to some native `fetch` improvements I did in `@readme/httpsnippet`, completely dropping support for funky `FormData` objects that `node-fetch@2` supports, the `convert` function there is now async.